### ISSUE TITLE
Unnecessary Doc

### ISSUE THREAD

**Issue by haiderabbas007** [ubsuny/CP1-24-HW2#88]
The following .md file is empty, might as well remove it.
![image](https://github.com/user-attachments/assets/7eab8289-8625-488c-8c8b-cbd4d0f44ae8)

**Comment by ojha-aditya** [comment]
Yeah, there are a couple of placeholder files, which are now safe to remove I guess, someone in need of a PR might take up the job. The blank Jupyter notebook and this.

**Comment by ojha-aditya** [comment]
The example notebook in discussion in #31

**Comment by haiderabbas007** [comment]
but there is code in it now
